### PR TITLE
Add MinIO File Upload & Download Example

### PR DESCRIPTION
**Pull request by itpey**
### Overview
Hello! 👋

This PR provides an example of file upload and download using Fiber with MinIO. It includes:

- An example for uploading and downloading a single file.
- An example for uploading multiple files.
- Step-by-step instructions for running each example.

This addition serves as a practical guide for developers to manage file storage with MinIO and Fiber.

## Summary by CodeRabbit

- **New Features**
	- Added a new "MinIO" section to the README for easier navigation.
	- Introduced a comprehensive README for a Go Fiber application showcasing file upload and download functionalities using MinIO.
	- Implemented web services for uploading and retrieving files with detailed error handling.

- **Documentation**
	- Enhanced documentation with prerequisites, project structure, and usage instructions for MinIO examples.

### CODE REVIEW

**Review by ReneWerner87 — APPROVED** [review]

**Comment by ReneWerner87** [comment]
@itpey thx for the implementation
Can you check the improvement hints from our ai

**Comment by itpey** [comment]
> @itpey thx for the implementation
> Can you check the improvement hints from our ai

I've reviewed the AI suggestions, implemented the important improvements, and committed the changes. Please let me know if there are any additional adjustments you'd like to see!

File: minio/go.mod [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module main
+
+go 1.22
+
+require (
+	github.com/gofiber/fiber/v2 v2.52.5
+	github.com/minio/minio-go/v7 v7.0.80
```
Comment (gaby) [inline minio/go.mod:7]: We already have an official Min.io client that would reduce this recipe boilerplate a lot. Why not use that?

https://github.com/gofiber/storage/tree/main/minio

**Comment by ReneWerner87** [comment]
i will check it again, tomorrow

File: minio/go.mod (hunk 1 above)
Comment (gaby) [reply minio/go.mod:7]: You can access the underlaying client by using `s.Conn()`

Comment (itpey) [reply minio/go.mod]: Yes, you are correct—I've replaced the MinIO client with Fiber's official MinIO storage to reduce the boilerplate. Thanks for the suggestion!

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**